client/src/components/Draftail/Draftail.scss

```diff
@@ -1,4 +1,4 @@
-$editor-z-index: $draftail-editor-z-index;
+$editor-z-index: 27;
```

Comment:
This used to be calculated based on another variable, which has been removed.

Reply:
Should we create new variable anyway?